php/libraries/User.class.inc

```diff
@@ -55,12 +55,28 @@ class User extends UserPermissions
         $DB =& Database::singleton();
 
         // get user data from database
-        $query = "SELECT users.*, psc.Name AS Site FROM users
-            LEFT JOIN psc ON (users.centerID=psc.CenterID)
-            WHERE users.UserID = :UID";
+        $query = "SELECT users.*,
+            GROUP_CONCAT(psc.Name SEPARATOR '; ') AS Site,
```

Comment:
changed to Sites (in User.class.inc and main.tpl); I am going through the rest of the scripts to see where else changes related to this might be needed.